test/helper.lua

```diff
@@ -4,15 +4,21 @@
 local fio = require('fio')
 local t = require('luatest')
 
-local helper = {}
+local ok, cartridge_helpers = pcall(require, 'cartridge.test-helpers')
+if not ok then
+    log.error('Please, install cartridge rock to run tests')
+    os.exit(1)
+end
 
-helper.root = fio.dirname(fio.abspath(package.search('init')))
-helper.datadir = fio.pathjoin(helper.root, 'tmp', 'db_test')
-helper.server_command = fio.pathjoin(helper.root, 'init.lua')
+local helpers = table.copy(cartridge_helpers)
+
+helpers.root = fio.dirname(fio.abspath(package.search('init')))
+helpers.datadir = fio.pathjoin(helpers.root, 'tmp', 'storage_test')
+helpers.server_command = fio.pathjoin(helpers.root, 'test', 'entrypoint', 'srv_kv_storage.lua')
```

Comment:
I think that it's not a good idea to have a single entrypoint and data directory for all tests.

I suggest using smth like https://github.com/tarantool/crud/blob/master/test/helper.lua#L36.